.github/workflows/terraform-plan-apply.yml

```diff
@@ -186,7 +186,7 @@ jobs:
           set -o errexit
           set -o nounset
 
-          wget https://github.com/NDPH-ARTS/mts-trial-ui/releases/download/latest/mts-trial-ui.zip
+          wget https://github.com/NDPH-ARTS/mts-trial-ui/releases/download/branding-assets/mts-trial-ui.zip
```

Comment:
How do we get the correct logo for a given trial deployment? This just grabs whatever was built.

Reply:
should be renamed to latest.
once the release is changed.